### PR TITLE
ES-v5.26-docs

### PR DESCRIPTION
**Pull request by esethna**

### CODE REVIEW

File: source/administration/config-settings.rst [hunk 1]
```diff
@@ -3711,7 +3711,7 @@ Experimental Sidebar Features (Experimental)
 
 **Disabled:** Users cannot access the experimental channel sidebar feature set.
 
-**Enabled (Default On):** Enables the experimental sidebar features for all users on this server. Users can disable the features in **Account Settings > Sidebar > Experimental Sidebar Features**. Features include collapsible categories, unread filtering, and history arrows. `Learn more or give us feedback <https://about.mattermost.com/default-sidebar/>`_.
+**Enabled (Default On):** Enables the experimental sidebar features for all users on this server. Users can disable the features in **Account Settings > Sidebar > Experimental Sidebar Features**. Features include custom collapsible channel categories, drag & drop to reorganize channels, and unread filtering. `Learn more or give us feedback <https://about.mattermost.com/default-sidebar/>`_.
```
Comment (amyblais) [inline source/administration/config-settings.rst:3714]:
```suggestion
**Enabled (Default On):** Enables the experimental sidebar features for all users on this server. Users can disable the features in **Account Settings > Sidebar > Experimental Sidebar Features**. Features include custom collapsible channel categories, drag and drop to reorganize channels, and unread filtering. `Learn more or give us feedback <https://about.mattermost.com/default-sidebar/>`_.
```

File: source/administration/config-settings.rst [hunk 2]
```diff
@@ -3793,6 +3793,14 @@ There are a number of settings customizable in ``config.json`` unavailable in th
 Service Settings
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
+Data Prefetch
+^^^^^^^^^^^^^^
+When enabled, messages in all unread channels are fetched from the server whenever the client reconnects to the network to eliminate loading time when users switch to unread channels.
```
Comment (amyblais) [inline source/administration/config-settings.rst:3798]: If it's true and false, we normally list them separately explaining what happens when it's true and when it's false.

File: source/help/messaging/formatting-text.rst [hunk 3]
```diff
@@ -179,7 +179,7 @@ Supported languages are:
 ``lua``, ``m``, ``mak``, ``matlab``, ``md``, ``mk``, ``mkd``, ``mkdown``,
 ``ml``, ``mm``, ``nc``, ``objc``, ``obj-c``, ``osascript``, ``pas``, ``pascal``,
 ``perl``, ``php``, ``php3``, ``php4``, ``php5``, ``php6``, ``pl``, ``plist``,
-``podspec``, ``ps``, ``ps1``, ``pp``, ``py``, ``r``, ``rb``,
+``podspec``, ``pgsql``, ``postgres``, ``postgresql``, ``ps``, ``ps1``, ``pp``, ``py``, ``r``, ``rb``,
```
Comment (amyblais) [inline source/help/messaging/formatting-text.rst:182]: These should be in alphabetical order

**Comment by esethna** [comment]
Updated

**Pull request merged**